includes/class-recaptcha.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/**
+ * Class for reCAPTCHA integration.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for reCAPTCHA integration.
+ */
+final class Recaptcha {
+	const RECAPTCHA_SCRIPT_HANDLE = 'newspack-recaptcha';
+	const RECAPTCHA_THRESHOLD     = 0.5;
```

Comment:
```suggestion
	const SCRIPT_HANDLE           = 'newspack-recaptcha';
	const SCORE_THRESHOLD         = 0.5;
```

Prefixing with `RECAPTCHA_` is a bit redundant inside the `\Newspack\Recaptcha` class.

Reply:
Renamed in 595102e (and https://github.com/Automattic/newspack-newsletters/pull/935/commits/00dbd1761e246f8cf3b672c47001ac1be3c75f4a and https://github.com/Automattic/newspack-blocks/pull/1244/commits/be6eafb5878e40750442b2a74bcbf423be12ac3f)